src/Parser/MarksmanshipHunter/Modules/Features/AlwaysBeCasting.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import CoreAlwaysBeCasting from 'Parser/Core/Modules/AlwaysBeCasting';
+
+import SPELLS from 'common/SPELLS';
+import Icon from 'common/Icon';
+import { formatPercentage } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import SpellLink from 'common/SpellLink';
+
+class AlwaysBeCasting extends CoreAlwaysBeCasting {
+  static ABILITIES_ON_GCD = [
+
+    // Marksmanship:
+    SPELLS.AIMED_SHOT.id,
+    SPELLS.WINDBURST.id,
+    SPELLS.ARCANE_SHOT.id,
+    SPELLS.MULTISHOT.id,
+    SPELLS.MARKED_SHOT.id,
+    SPELLS.TAR_TRAP.id,
+    SPELLS.FREEZING_TRAP.id,
+
+    SPELLS.BLACK_ARROW_TALENT.id,
+    SPELLS.EXPLOSIVE_SHOT_TALENT.id,
+    SPELLS.SIDEWINDERS_TALENT.id,
+    SPELLS.PIERCING_SHOT_TALENT.id,
+    SPELLS.TRICK_SHOT_TALENT.id,
+    SPELLS.A_MURDER_OF_CROWS_TALENT_SHARED.id,
+    SPELLS.BARRAGE_TALENT.id,
+
+
+  ];
+
+  suggestions(when) {
+    const deadTimePercentage = this.totalTimeWasted / this.owner.fightDuration;
+
+    when(deadTimePercentage).isGreaterThan(0.2)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span>Your dead GCD time can be improved. Try to Always Be Casting (ABC), try to reduce the delay between casting spells. Even if you have to move, try casting something instant with range like <SpellLink id={SPELLS.FLAMETONGUE.id}/> or <SpellLink id={SPELLS.ROCKBITER.id}/></span>)
```

Comment:
These don't seem like things a Hunter can do. ;) What *Hunter* spells would you recommend casting while moving?

Reply:
yeah, used some of nighteyz code as a template and missed this. Will correct in next pull request